aspnetcore-webapi/IgnorePropertyInSwagger/IgnorePropertyInSwagger/SchemaFilters/SwaggerIgnoreFilter.cs

```diff
@@ -0,0 +1,29 @@
+using System.Reflection;
+using System.Text.Json.Serialization;
+using IgnorePropertyInSwagger.Attributes;
+using IgnorePropertyInSwagger.Extensions;
+using Microsoft.OpenApi.Models;
+using Swashbuckle.AspNetCore.SwaggerGen;
+
+namespace IgnorePropertyInSwagger.SchemaFilters;
+
+public class SwaggerIgnoreFilter : ISchemaFilter
+{
+    public void Apply(OpenApiSchema schema, SchemaFilterContext schemaFilterContext)
+    {
+        if (schema.Properties.Count == 0)
+            return;
```

Comment:
Always use curly braces even though there is only one line inside the `if` statement

Reply:
done